force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -110,7 +155,17 @@ public with sharing class FlowControlLoopsCollections {
      * @return The grade for the given score.
      */
     public static String determineGrade(Integer score) {
-        return null; // Replace null with the variable you used to store the result
+        if(score >= 90){
```

Comment:
Well done arranging the logic. Also, the formatting is better here! I would suggest adding a space around the brackets and parentheses. Like this: `if (score >= 90) {`